File: WebApiDotNetCoreBackEnd/Common/ViewModels/UserViewModel.cs
using Model;
using System;
using System.Collections.Generic;
using System.Text;
using System.Linq;

namespace Common.ViewModels
{
    public class UserViewModel
    {
        public long IdUser { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public string Password { get; set; }
        public bool Active { get; set; }
        public string RolesList { get; set; }
        public bool UpdatePassword { get; set; }

        public List<RoleViewModel> Roles { get; set; }

    }
}

File: AngularFrontEnd/src/app/models/UserViewModel.ts
import { ResolveStart } from '@angular/router';
import { RoleViewModel } from './RoleViewModel';

export class UserViewModel{
    idUser: number;
    name: string;
    email: string;
    password: string;
    active: boolean;
    rolesList: string;
    updatePassword: boolean;
    roles: RoleViewModel[];
    
    constructor(){
        this.idUser = 0;
    }  
}